### ISSUE TITLE
Use this after cursor selection position is out of order

### ISSUE THREAD

**Issue by meikang123** [josdejong/jsoneditor#1403]
![image](https://user-images.githubusercontent.com/16129828/146911389-7a57a610-cc86-43c4-a4e7-c1f0561c443d.png)

**Comment by meikang123** [comment]
![image](https://user-images.githubusercontent.com/16129828/146911852-cf93f9c9-8f10-4056-ae98-1509a4b52caa.png)
![image](https://user-images.githubusercontent.com/16129828/146911904-3411825f-62a5-43e5-b0e7-f7254205a936.png)
Choosing to modify is problematic

**Comment by widavies** [comment]
Yeah I was having the same problem in `code` mode. For some reason the cursor position did not match up with where text was being inserted. To fix this, I applied the CSS:
```css
.ace-jsoneditor *, textarea.jsoneditor-text * {
    font-family: unset;
}
```

**Comment by meikang123** [comment]
> Yeah I was having the same problem in `code` mode. For some reason the cursor position did not match up with where text was being inserted. To fix this, I applied the CSS:
> 
> ```css
> .ace-jsoneditor *, textarea.jsoneditor-text * {
>     font-family: unset;
> }
> ```

It's just that this font isn't as good looking as the one above

**Comment by josdejong** [comment]
Can it be related to no charset being defined on the page? ie. `<meta charset="utf-8">` missing in the html page? (see #1342)

Does this also occur with `code` mode on the website https://jsoneditoronline.org?

**Comment by meikang123** [comment]
> Can it be related to no charset being defined on the page? ie. `<meta charset="utf-8">` missing in the html page? (see #1342)
> 
> Does this also occur with `code` mode on the website https://jsoneditoronline.org?

![image](https://user-images.githubusercontent.com/16129828/147113478-364ccea6-ce0a-47a3-a56d-0cbb000a4517.png)
It shouldn't be missing

**Comment by josdejong** [comment]
I've done some experiments. It looks like this issue is introduced in the last version 9.5.8 where we changed the font-family and font-size. The current font on Windows doesn't work nicely in code mode (Ace editor), the caret has some offset.

**Comment by josdejong** [comment]
I've just published `v9.5.9` containing fixes for the font, you can also try it out on https://jsoneditoronline.org.

Does that solve the issue for you?

**Comment by meikang123** [comment]
> I've just published `v9.5.9` containing fixes for the font, you can also try it out on https://jsoneditoronline.org.
> 
> Does that solve the issue for you?

Thank you. Problem solved

**Comment by josdejong** [comment]
🎉  thanks for the feedback (and for reporting the issue in the first place)
